package.json

```diff
@@ -49,9 +49,6 @@
       "immer@8": "10.1.3",
       "react-refresh": "0.14.0"
     },
-    "patchedDependencies": {
-      "jest-resolve@26.6.2": "patches/jest-resolve@26.6.2.patch"
```

Comment:
The [jest-resolve@26.6.2.patch](https://github.com/plone/volto/blob/main/patches/jest-resolve%4026.6.2.patch) file can be removed.